public/app/types/datasources.ts

```diff
@@ -15,6 +16,14 @@ export interface DataSourcesState {
   categories: DataSourcePluginCategory[];
 }
 
+export interface DataSourceSettingsState {
+  plugin?: GenericDataSourcePlugin;
+  isTesting?: boolean;
```

Comment:
I just realized that I can't find any usage of `isTesting`.

Reply:
You're correct. It has never been used.